packages/cron/package.json

```
@@ -0,0 +1,71 @@
+{
+	"name": "@sapphire/cron",
+	"version": "0.0.0",
```

```suggestion
	"version": "1.0.0",
```

No more lerna = no more hacky first version